examples/counter/test/sagas.js

```diff
@@ -25,5 +26,20 @@ test('incrementAsync Saga test', (t) => {
     'counter Saga must be done'
   )
 
+  const watcher = rootSaga()
+  const worker = takeEvery('INCREMENT_ASYNC', incrementAsync);
+
+  t.deepEqual(
+    watcher.next().value,
+    worker.next().value,
+    'rootSaga takes from every INCREMENT_ASYNC action'
+  )
+
+  t.equal(
+    watcher.return().done,
```

Comment:
Am I missing something? I went in to update this and the tests are still passing. It looks like takeEvery is still returning an iterator. Do I have something wrong??